when-shit-hits-the-fan.adoc

```diff
@@ -332,9 +332,9 @@ The "value out" in this block #74638 is quite strange:
 92233720368.54277039 BTC?  Is that UINT64_MAX, I wonder?
 ____
 
-Apparently, the two int64 (not uint64 as Garzik pondered) outputs' sum
+Apparently, the two int64 (not uint64, as Garzik supposed) outputs' sum
 would overflow to a negative value -0.00997538 BTC. Whatever the sum
-of the inputs, the "sum" of the outputs will be smaller, making
+of the inputs, the "`sum`" of the outputs would be smaller, making
 this transaction OK according to the code at the time.
```

Comment:
Add this sentence (or something similar)

"The bug had been disclosed and published through an actual exploit."

Reply:
Ok, I integrated it and it sounds alright to me. Just let me know if it doesn't convince you and we'll find a new form